docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -30,22 +30,32 @@ Array.sum array
 ```
 
 #### Parameters
-*array*
-Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
-
+*array* Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
 
 The input array.
 
 
+## Return Value
+
+The resulting sum.
+
 
-**The resulting sum.**
 ## Remarks
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception(v=vs.110).aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
```

Comment:
This should link to "https://msdn.microsoft.com/en-us/library/system.overflowexception" (without the version tags, to avoid specific versions of OverflowException being stale as things move forward)